### PR TITLE
Add support for LTWA (List of Title Word Abbreviations)

### PR DESCRIPTION
**Pull request by wenshilanR**
Resolves https://github.com/koppor/jabref/issues/215

The current problem is that the matcher is design to match the whole title.

The matcher has to be rewritten to match single words.

### Mandatory checks
- [ ] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [ ] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [ ] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

**Comment by ThiloteE** [comment]
To ease organizational workflows I have linked the pull-request to the issue with syntax as described in https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

> <h2 id="linking-a-pull-request-to-an-issue-using-a-keyword">Linking a pull request to an issue using a keyword</h2>
> <p>You can link a pull request to an issue by using a supported keyword 
> in the pull request's description or in a commit message. The pull 
> request <strong>must be</strong> on the default branch.</p>
> <ul>
> <li>close</li>
> <li>closes</li>
> <li>closed</li>
> <li>fix</li>
> <li>fixes</li>
> <li>fixed</li>
> <li>resolve</li>
> <li>resolves</li>
> <li>resolved</li>
> </ul>
> <p>If you use a keyword to reference a pull request comment in another 
> pull request, the pull requests will be linked. Merging the referencing 
> pull request also closes the referenced pull request.</p>
> <p>The syntax for closing keywords depends on whether the issue is in the same repository as the pull request.</p>

**Comment by Siedlerchr** [comment]
I don't see anything except a whitespace change? Did you push your changes?

File: src/main/java/org/jabref/gui/preferences/journals/JournalAbbreviationsTabViewModel.java [hunk 1]
```diff
@@ -230,7 +230,7 @@ public void addAbbreviation(Abbreviation abbreviationObject) {
     public void addAbbreviation() {
         addAbbreviation(new Abbreviation(
                 Localization.lang("Name"),
-                Localization.lang("Abbreviation"),
+                Localization.lang("abbreviation"),
```
Comment (Siedlerchr) [inline src/main/java/org/jabref/gui/preferences/journals/JournalAbbreviationsTabViewModel.java:233]: Please revert the change here

Comment (wenshilanR) [reply src/main/java/org/jabref/gui/preferences/journals/JournalAbbreviationsTabViewModel.java:233]: Certainly! I think I accidentally made this change, sorry.

File: src/main/java/org/jabref/logic/journals/JournalAbbreviationLoader.java [hunk 2]
```diff
@@ -40,6 +40,7 @@ public static JournalAbbreviationRepository loadRepository(JournalAbbreviationPr
                 LOGGER.warn("There is no journal-list.mv. We use a default journal list");
                 repository = new JournalAbbreviationRepository();
             } else {
+
```
Comment (koppor) [inline src/main/java/org/jabref/logic/journals/JournalAbbreviationLoader.java:43]: No unnecesary newline changes please

**Review by koppor — COMMENTED** [review]
Still on early stage this PR. No MV file handling etc.

**Comment by koppor** [comment]
Since there was no activity for a long time, I'll close this PR. Feel free to open a new PR in case you resume work on this.
